lib/models/order.js

```diff
@@ -329,7 +329,7 @@ class Order extends Model {
 Order.type = {}
 Order.status = {}
 
-const statuses = ['ACTIVE', 'EXECUTED', 'PARTIALLY FILLED', 'CANCELLED']
+const statuses = ['ACTIVE', 'EXECUTED', 'PARTIALLY FILLED', 'CANCELED']
```

Comment:
same typo in other models?

example:

https://github.com/bitfinexcom/bitfinex-api-node/blob/2ebbe05f8c6bcec1d03c9eca99258be0f476678e/lib/models/funding_loan.js#L44